test/functional/utils.js

```diff
@@ -32,11 +37,11 @@ const FIREFOX_PREFERENCES = {
 };
 
 // Re-usable test methods from shield-studies-addon-utils
-const { executeJs } = require("../../../../testUtils/executeJs");
-const { nav } = require("../../../../testUtils/nav");
-const { setup } = require("../../../../testUtils/setup");
-const { telemetry } = require("../../../../testUtils/telemetry");
-const { ui } = require("../../../../testUtils/ui");
+const { executeJs } = require("../../testUtils/executeJs");
+const { nav } = require("../../testUtils/nav");
+const { setup } = require("../../testUtils/setupWebdriver");
```

Comment:
Inconcistency between exported variable and filename. Also, WebDriver is cased as WebDriver in comments earlier in this file.